autometrics-macros/src/lib.rs

```diff
@@ -112,6 +112,10 @@ fn instrument_function(args: &AutometricsArgs, item: ItemFn) -> Result<TokenStre
         quote! { None }
     };
 
+    let service_name = quote! {
+        autometrics::__private::service_name(env!("CARGO_PKG_NAME"))
```

Comment:
If my service name is set via a config file / flag instead of an environment variable, is there a plan for supporting that?

Reply:
Good question. There is a way we could support that, I'm just not sure if we should do it now. I started a separate issue to discuss this https://github.com/autometrics-dev/autometrics-rs/issues/117